etl/steps/data/garden/dummy/2020-01-01/dummy_full.meta.yml

```diff
@@ -4,14 +4,14 @@ tables:
       dummy_variable:
         title: "[indicator.title]"
         unit: "[indicator.unit]"
-        short_unit: "[indicator.short-unit]"
+        short_unit: "[indicator.short_unit]"
```

Comment:
I changed the `_` by `-` so that in data pages the text between underscores would not be parsed as italics. Have you figured out how to avoid it? Otherwise I think it's better to stick to `-`.

Reply:
Oh, I didn't realise that 🤦 reverted it.